samples/cpp/peopledetect.cpp

```diff
@@ -5,7 +5,6 @@
 #include <opencv2/objdetect.hpp>
 #include <opencv2/highgui.hpp>
 #include <opencv2/imgproc.hpp>
-#include <opencv2/videoio.hpp>
```

Comment:
Probably we should keep separate include of "videoio" module - `VideoCapture` is used below.
(due compatibility "videoio" headers are included by "highgui" header, but this doesn't look consistent - lets show to Users which modules are used)

Reply:
@alalek OK. let me add `#include <opencv2/videoio.hpp>` in all samples uses `VideoCapture`
please don't merge until i inform you.